src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,118 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int OFFSET = 1;
+    private static final int START_INDEX = 0;
+    private static final int INITIAL_CAPACITY = 10;
+    private static final double ELEMENT_DATA_MULTIPLIER = 1.5;
+    private int listCapacity;
```

Comment:
this is a useless field, we can always get capacity by calling .length on our data container (T[])